js/data/move_data.js

```diff
@@ -3038,7 +3047,7 @@ var MOVES_SM = $.extend(true, {}, MOVES_XY, {
     'Outrage': { zp: 190 },
     'Overheat': { zp: 195 },
     'Paleo Wave': { zp: 160 },
-    'Parabolic Charge': { bp: 65, zp:120 },
+    'Parabolic Charge' : { bp: 65, zp:120 },
```

Comment:
Also what is this

Reply:
Alright, so after double checking you were right throughout the whole process... I just am used to type that kind of stuff this way, space before the :, including other things you were and you are pointing at as formatting issues. I apologize.

Reply:
It’s fine, it happens all the time